feature/pseudonyms-part-5

Figments opened the pull request:
this PR removes portfolios and migrates them to profiles, encapsulated in their own module.

Figments committed:
adding profile module

Figments committed:
Merge branch 'main' into feature/pseudonyms-part-5

Figments committed:
adding routing

Figments committed:
fixing profile header

Figments committed:
Merge branch 'feature/pseudonyms-part-5' of https://github.com/OffprintStudios/dragonfish into feature/pseudonyms-part-5

Figments committed:
add boolean check for menu opened

Figments committed:
adding links to profile, adding followers count to header

Figments committed:
works page is back

Figments committed:
finished header styling for now

Figments committed:
add blog page skeleton file

Figments committed:
blogs feed

Figments committed:
remove old portfolio routes

Figments committed:
Merge branch 'main' into feature/pseudonyms-part-5

SigmasonicX reviewed (changes requested):
Code-wise, this seems fine, but the profile header and its animations don't look good on my laptop.

The animations for going between topbar modes looks bad.

The top is cut off
![image](https://user-images.githubusercontent.com/71996084/132299205-93c865ac-ef17-4c7f-a912-2ffd11f70af9.png)

It takes up a lot of screen real estate when scrolling down
![image](https://user-images.githubusercontent.com/71996084/132299268-66768088-4ce6-4cd0-b357-216841e58bb7.png)

And we should consider how it looks on mobile
![image](https://user-images.githubusercontent.com/71996084/132299359-eaf9ecd2-d538-440f-9a8e-fbdf821ac021.png)

Figments committed:
applied fixes

- fixed scrolling
- removed animations for topbar changes
- removed topbar changes
- fixed avatar cutting off when no cover pic

Figments committed:
Merge branch 'main' into feature/pseudonyms-part-5

Figments committed:
fixed performance issues on home page

SigmasonicX reviewed (approved):
LGTM
